pkg/ingester/ingester_v2.go

```diff
@@ -898,6 +953,7 @@ func (i *Ingester) createTSDB(userID string) (*userTSDB, error) {
 	// Run compaction before using this TSDB. If there is data in head that needs to be put into blocks,
 	// this will actually create the blocks. If there is no data (empty TSDB), this is a no-op, although
 	// local blocks compaction may still take place if configured.
+	level.Info(userLogger).Log("msg", "Running compaction after WAL replay")
```

Comment:
Add the user ID because we may have multiple concurrent tenants replaying WAL.

```suggestion
	level.Info(userLogger).Log("msg", "running compaction after WAL replay", "user", userID)
```

Reply:
We already use `userLogger` for that purpose.